src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -535,15 +557,99 @@ def _visit_neighbors(self, node: gtir.FunCall) -> DataExpr:
+                if set(input_expr.subset.size()[:-1]) != {1}:
+                    raise ValueError(f"Invalid node {node}")
+                rstart, rstop, rstep = input_expr.subset[-1]
+                assert rstart == 0 and rstep == 1
+
+                desc = self.sdfg.arrays[input_expr.node.data]
+                view, _ = self.sdfg.add_view(
+                    f"{input_expr.node.data}_view",
+                    (rstop + 1,),
+                    desc.dtype,
+                    strides=desc.strides[-1:],
+                    find_new_name=True,
+                )
+                input_node = self.state.add_access(view)
+                self._add_input_data_edge(input_expr.node, input_expr.subset, input_node)
+
+            else:
+                # this is the case of scalar value broadcasted on a list by make_const_list
+                assert isinstance(input_expr, DataExpr)
+                input_node = input_expr.node
+
+            assert len(input_node.desc(self.sdfg).shape) == 1
+            input_size = input_node.desc(self.sdfg).shape[0]
+            if input_size == 1:
+                input_memlets[conn] = dace.Memlet(data=input_node.data, subset="0")
+            else:
+                input_memlets[conn] = dace.Memlet(data=input_node.data, subset=map_index)
+                if local_size and input_size != local_size:
+                    raise ValueError(f"Invalid node {node}")
+                else:
+                    local_size = input_size
+
+            input_nodes[input_node.data] = input_node
+
+        if local_size is None:
+            # corner case where map is applied to 1-element lists
+            assert len(input_nodes) >= 1
+            local_size = 1
+
+        out, _ = self.sdfg.add_temp_transient((local_size,), dtype)
+        out_node = self.state.add_access(out)
+
+        assert isinstance(node.fun, gtir.FunCall)
+        assert len(node.fun.args) == 1
+        fun_node = im.call(node.fun.args[0])(*connectors)
+        op_code = gtir_python_codegen.get_source(fun_node)
+
+        self._add_mapped_tasklet(
+            name="map",
+            map_ranges={map_index: f"0:{local_size}"},
+            code=f"__out = {op_code}",
+            inputs=input_memlets,
+            input_nodes=input_nodes,
+            outputs={
+                "__out": dace.Memlet(data=out, subset=map_index),
+            },
+            output_nodes={out: out_node},
+            external_edges=True,
+        )
+
+        return DataExpr(out_node, dtype)
+
     def _visit_reduce(self, node: gtir.FunCall) -> DataExpr:
         op_name, reduce_init, reduce_identity = get_reduce_params(node)
         dtype = reduce_identity.dtype
 
         # We store the value of reduce identity in the visitor context while visiting
-        # the input to reduction; this value will be use by the `neighbors` visitor
+        # the input to reduction; this value will be used by the `neighbors` visitor
```

Comment:
Shouldn't it be "might be used by"?
Also where does the `neighbors` built in comes from?